doc/nrf/device_guides/working_with_nrf/nrf91/nrf9161.rst

```diff
@@ -0,0 +1,319 @@
+.. _ug_nrf9161:
+
+Developing with nRF9161 DK
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF9161 DK is a hardware development platform used to design and develop application firmware on the nRF9161 :term:`System in Package (SiP)`.
+The nRF9161 SiP is a low power SiP with integrated DECT NR+ modem and 3GPP Release 14 LTE-M/NB-IoT with GNSS.
+
+See the nRF9161 DK Hardware guide for detailed information about the nRF9161 DK hardware.
+To get started with the nRF9161 DK, follow the steps in the :ref:`ug_nrf9161_gs` section.
+If you are not familiar with the |NCS|, see :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
+
+.. _nrf9161_ug_intro:
+
+Board controller
+****************
+
+
+
+.. _nrf9161_ug_updating_cloud_certificate:
+
+Updating the nRF Cloud certificate
+**********************************
+
+.. |DK| replace:: nRF9161 DK
```

Comment:
Not sure if it's needed, but looks like `|DK|` could be used a few more places in this file

Reply:
Yeah, I just meant that there are a few occurrences of `nRF9161 DK` in the file instead of using `|DK|`, but it's not a big deal